transcom/mymove
# B 20154 int UI display PPM shipment address on review documents page

## [B-20154](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A958127)

## Summary

This backlog gets rid of the display of PPM pickup and destination zip codes with the full addresses on the Service Counselor's Review Documents page for PPM shipments.

[this article](tbd) explains more about the approach used.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Login as a Customer
2. Create and submit a PPM shipment
3. Login as a Service Counselor and approve the PPM shipment
4. Login as the Customer and upload PPM documents
5. Submit the documents
6. Login as the Service Counselor and go to the Closeout Queue
7. Click Review Documents for the PPM shipment
8. Expand the PPM details section and verify the full Pickup and Destination Addresses are listed

### Frontend
